tests/python/unittest/test_numpy_op.py

```diff
@@ -9244,21 +9243,24 @@ def hybrid_forward(self, F, a):
 
     rtol = 1e-2 if dtype == np.float16 else 1e-3
     atol = 1e-4 if dtype == np.float16 else 1e-5
-    shape = config[0]
-    axis = config[1]
-    axis1 = axis[0]
-    axis2 = axis[1]
+    shape, (axis1, axis2) = config
     x = np.random.uniform(-5.0, 5.0, size=shape).astype(dtype)
     x.attach_grad()
     test_diagonal = TestDiagonal(k, axis1, axis2)
```

Comment:
The test case you modified did not cover the hybridize case (i.e. the symbol mode), so I suggest changing the function call at https://github.com/apache/incubator-mxnet/blob/master/tests/python/unittest/test_numpy_op.py#L9243 as well to test if your patch is fully compatible with hybrid block.

Reply:
@xidulu Thanks for the review. I’ve added two new commits https://github.com/apache/incubator-mxnet/pull/19285/commits/c9c8aa7df44c7936c4e54c1716ac1d5cac6c2f4a and https://github.com/apache/incubator-mxnet/pull/19285/commits/8ae59cefad67fa1eea1cc5a6bc0033dc6faa570a to cover the hybridize case.